pages/api/v1/teams/[id]/roster/stats.ts

```diff
@@ -17,7 +20,7 @@ export default async (
   const {
     id,
     league = 0,
-    type: shorttype = 'rs',
+    SeasonType,
```

Comment:
Can we keep the `type` parameter instead of the `SeasonType` parameter? SeasonType doesn't follow our conventions that are in place for query param names. We can instead do something like:


```suggestion
    type: longType =  'Regular',
```